plugin/setup_test.go

```diff
@@ -0,0 +1,21 @@
+package lighthouse
```

Comment:
you need to adapt to use ginkgo/gomega.

Reply:
Will do in subsequent patches. Still figuring out how to write tests with ginkgo